### PR TITLE
[Merged by Bors] - chore(SetTheory/Game/PGame): deprecate primed lemmas

### PR DESCRIPTION
**Pull request by vihdzp**
We perform various renames with the intent of doing away with some badly named or dubiously useful lemmas.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by YaelDillies — APPROVED** [review]
Thanks! 🚀

maintainer merge

**Comment by b-mehta** [comment]
If I'm understanding correctly, you're not actually removing any lemmas here, right? Just renaming them to make a bit more sense

**Comment by vihdzp** [comment]
I'm renaming some, but also deprecating some like `moveLeft_neg_symm` which are just compositions of previous lemmas which `simp` can prove anyways.

File: Mathlib/SetTheory/Game/PGame.lean [hunk 1]
```diff
@@ -1149,34 +1149,40 @@ between them. -/
...
   cases x
   rfl
 
-theorem moveRight_neg {x : PGame} (i) : (-x).moveRight (toRightMovesNeg i) = -x.moveLeft i := by
-  cases x
-  rfl
+@[deprecated moveLeft_neg (since := "2024-10-30")]
+alias moveLeft_neg' := moveLeft_neg
+
+theorem moveLeft_neg_toLeftMovesNeg {x : PGame} (i) :
```
Comment (b-mehta) [inline Mathlib/SetTheory/Game/PGame.lean:1161]: It looks like the right version of this is now missing: `(-x).moveRight (toRightMovesNeg i) = -x.moveLeft i`, which is useful as a rewrite lemma

**Review by b-mehta — APPROVED** [review]
Ah okay, I see. I agree with most of the deprecations, but there's one which should be there

**Review by j-loreaux — APPROVED** [review]
Thanks!

bors d+

**Pull request closed without merging**